From 381e82dc6f28448642299a073b60946fe2f114b3 Mon Sep 17 00:00:00 2001
From: Mikolaj Izdebski <mizdebsk@redhat.com>
Date: Tue, 12 May 2015 10:02:34 +0200
Subject: [PATCH 4/4] Fix hardlink creation

---
 java-utils/java-functions | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/java-utils/java-functions b/java-utils/java-functions
index dc8aa1b..c827042 100644
--- a/java-utils/java-functions
+++ b/java-utils/java-functions
@@ -333,7 +333,7 @@ link_jar_repository() {
    while [ $# -gt 0 ] ; do
       case "$1" in
          -h|--hard)
-            _LINK_CMD="ln -f"
+            _LINK_CMD="ln -fL"
             ;;
          -s|--soft|--symbolic)
             _LINK_CMD="ln -fs"
-- 
2.1.0

